src/main/java/net/vulkanmod/vulkan/Drawer.java

```diff
@@ -25,17 +26,23 @@ public class Drawer {
     private int framesNum;
     private VertexBuffer[] vertexBuffers;
     private final AutoIndexBuffer quadsIndexBuffer;
+    private final AutoIndexBuffer linesIndexBuffer;
     private final AutoIndexBuffer triangleFanIndexBuffer;
-    private final AutoIndexBuffer triangleStripIndexBuffer;
+    private final AutoIndexBuffer stripIndexBuffer;
+    private final AutoIndexBuffer debugLinesIndexBuffer;
+    private final AutoIndexBuffer lineStripIndexBuffer;
     private UniformBuffer[] uniformBuffers;
 
     private int currentFrame;
 
     public Drawer() {
         //Index buffers
         quadsIndexBuffer = new AutoIndexBuffer(UINT16_INDEX_MAX, AutoIndexBuffer.DrawType.QUADS);
```

Comment:
_(This is complete nitpick, so feel free to skip this if you prefer)_

`UINT16_INDEX_MAX` should be 65536 instead of 98304: as Indices > 65536 overflow as a result of using `VK_INDEX_TYPE_UINT16`
_(Minecraft uses 16-bit instead of 32-Bit indices as they can have performance advantages)_

Which causes the index Buffers to be about 33% larger than needed.

This is my mistake as I added this `UINT16_INDEX_MAX` constant in an older VulkanMod version, but never corrected it, so that's MB.

Put a [small commit here](https://github.com/thr3343/VulkanMod/commit/730c66fe8d95958e66084a81393e29e76f4814db) w/ this optimization in case it helps as a reference.

Reply:
65536 / 4 * 6 > 65536